app/src/main/res/layout/checkbox_blinds.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_gravity="center_horizontal">
+
+
+    <CheckBox
```

Comment:
I like the idea of using checkboxes, but this should be more generic, allowing more than just bedroom, living room or kitchen.
Ideally the user should see all blinds in a list (quite similar to the `RecyclerView` in `fragment_blinds_settings.xml`, but with checkboxes for each of the items in the `RecyclerView`). Then, they are free to select any of their blinds individually.

Reply:
It seems that this was left over from him testing things out, and Jeremy wasn't able to figure out a way to do it. I'm doing this properly along with fixing and implementing other things on that page.